web_src/js/htmx.js

```diff
@@ -19,3 +20,15 @@ document.body.addEventListener('htmx:responseError', (event) => {
   // TODO: add translations
   showErrorToast(`Error ${event.detail.xhr.status} when calling ${event.detail.requestConfig.path}`);
 });
+
+// eslint-disable-next-line no-import-assign
+htmx.createWebSocket = (url) => {
+  // TODO: reuse websocket from shared webworker
+  const sock = new WebSocket(url, []);
+  sock.binaryType = htmx.config.wsBinaryType;
+  return sock;
+};
+document.body.addEventListener('htmx:wsOpen', (evt) => {
+  const socket = evt.detail.socketWrapper;
```

Comment:
```suggestion
document.body.addEventListener('htmx:wsOpen', (e) => {
  const socket = e.detail.socketWrapper;
```
We prefer `e` name for events.

Reply:
renamed